style-dictionary/visual-refresh/metadata/colors.ts

```diff
@@ -299,6 +304,12 @@ const metadata: StyleDictionary.MetadataIndex = {
     public: true,
     themeable: true,
   },
+  colorBorderSelectableItemFocused: {
+    description:
+      'The color of focus states for dropdown items, for example, the focus ring around selectable elements in the dropdown of button dropdown, select, and multi-select.',
```

Comment:
I would use a full stop instead of comma before "for example", like in the previous token descriptions: "The color of focus states for dropdown items. For example, ..."

Reply:
Agreed! Will change it